utils/releaser/src/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,169 @@
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface;
+use Symplify\MonorepoBuilder\Release\Guard\ReleaseGuard;
+use Symplify\MonorepoBuilder\Release\ValueObject\StaticSemVersion;
+use Symplify\MonorepoBuilder\Release\Version\VersionFactory;
+use Symplify\MonorepoBuilder\ValueObject\Option;
+use Symplify\PackageBuilder\Console\Command\CommandNaming;
+use Symplify\PackageBuilder\Console\ShellCode;
+
+final class ReleaseCommand extends Command
+{
+    private const RESUME_STEP = 'resume-step';
+
+    /**
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    /**
+     * @var \Symplify\MonorepoBuilder\Release\Guard\ReleaseGuard
+     */
+    private $releaseGuard;
+
+    /**
+     * @var \Shopsys\Releaser\ReleaseWorker\ReleaseWorkerProvider
+     */
+    private $releaseWorkerProvider;
+
+    /**
+     * @var \Symplify\MonorepoBuilder\Release\Version\VersionFactory
+     */
+    private $versionFactory;
+
+    /**
+     * @param \Symfony\Component\Console\Style\SymfonyStyle $symfonyStyle
+     * @param \Shopsys\Releaser\ReleaseWorker\ReleaseWorkerProvider $releaseWorkerProvider
+     * @param \Symplify\MonorepoBuilder\Release\Guard\ReleaseGuard $releaseGuard
+     * @param \Symplify\MonorepoBuilder\Release\Version\VersionFactory $versionFactory
+     */
+    public function __construct(
+        SymfonyStyle $symfonyStyle,
+        ReleaseWorkerProvider $releaseWorkerProvider,
+        ReleaseGuard $releaseGuard,
+        VersionFactory $versionFactory
+    ) {
+        parent::__construct();
+
+        $this->symfonyStyle = $symfonyStyle;
+        $this->releaseGuard = $releaseGuard;
+        $this->releaseWorkerProvider = $releaseWorkerProvider;
+        $this->versionFactory = $versionFactory;
+    }
+
+    protected function configure(): void
+    {
+        $this->setName(CommandNaming::classToName(self::class));
+        $this->setDescription('Perform release process with set Release Workers.');
+
+        $description = sprintf(
+            'Release version, in format "<major>.<minor>.<patch>" or "v<major>.<minor>.<patch> or one of keywords: "%s"',
+            implode('", "', StaticSemVersion::getAll())
+        );
+        $this->addArgument(Option::VERSION, InputArgument::REQUIRED, $description);
+
+        $this->addOption(
+            Option::DRY_RUN,
+            null,
+            InputOption::VALUE_NONE,
+            'Do not perform operations, just their preview'
+        );
+
+        $this->addOption(Option::STAGE, null, InputOption::VALUE_REQUIRED, 'Name of stage to perform');
+        $this->addOption(self::RESUME_STEP, null, InputOption::VALUE_OPTIONAL, 'Number of step to start from');
```

Comment:
_nitpick 🤓_

Shouldn't be value required? It does not make sense to run release as 
`php ./vendor/bin/monorepo-builder release v7.3.6 --stage release-candidate --resume-step`

Reply:
fixed